index.js

```diff
@@ -379,6 +381,26 @@ Tree.prototype._slide = function (source) {
 }
 
 /*
+ * Sets the `tree-overflow` class on the tree node based on `visibleNodes` length.
+ * `visibleNodes` is optional. If it's not received, we look up how many nodes are visible.
+ * But for performance reasons we allow an incoming argument to control how
+ * many nodes are visible in case this number is already known.
+ */
+Tree.prototype.overflowClassed = function (visibleNodes) {
```

Comment:
I coded it so SB3 doesn't need to worry about the visibleNodes. I only added it as an argument so I could pass in the number of visibleNodes each time we're rebinding data (such as expanding, collapsing, etc.). You can see there here: https://github.com/SpiderStrategies/kalpa-tree/blob/90ffeb1fdfe0aa9ed4f2b68f0183e2a371ed4e04/index.js#L321

When someone resizes the browser (or one of the panes/modals holding a tree) in SB3, you can call this method and ignore visibleNodes. It will use d3 to determine how many are in the tree.